Playbooks/playbook-Detonate_URL_-_Lastline.yml

```diff
@@ -0,0 +1,341 @@
+id: Detonate URL - Lastline
+version: -1
+name: Detonate URL - Lastline
+fromversion: 4.0.0
+description: Detonates a URL using the Lastline sandbox integration.
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: d2f4d7dc-4562-4517-8d63-3539676ca81d
+    type: start
+    task:
+      id: d2f4d7dc-4562-4517-8d63-3539676ca81d
+      version: -1
+      name: ""
+      description: ""
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "9"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 275,
+          "y": 50
+        }
+      }
+    note: false
+  "1":
```

Comment:
This task does not output context data correctly:
![447 sdfdsfeaf - workplan 2018-09-17 18-41-17](https://user-images.githubusercontent.com/39116813/45633828-86e20980-baa9-11e8-94ff-ecaec805ef8f.jpg)

Reply:
added relevant output fields in integration